### DISCUSSION TITLE
Streaming subscription for all folders

### DISCUSSION THREAD

**Discussion by CubePirate** [ecederstrand/exchangelib#1242] [Q&A] [answered]
I can't seem to figure out how to create a stream subscription for all folders, does the library support it?
I know EWS does support it, via "`<StreamingSubscriptionRequest SubscribeToAllFolders="">`"
See https://learn.microsoft.com/en-us/exchange/client-developer/web-service-reference/streamingsubscriptionrequest

**Comment by ecederstrand** [comment, marked as answer]
No, exchangelib does not support that option, but it would be a good addition, IMHO. I didn't know about it, so thanks for the heads-up!

**Reply by ecederstrand** [reply]
Reported and fixed in https://github.com/ecederstrand/exchangelib/issues/1243
